src/DrawTools/AtomTool.ts

```diff
@@ -36,8 +35,8 @@ let wasOut: boolean;
 //True if the mouse button is currently down.
 let hasMouseDown: boolean;
 
-//AtomNode we are creating. Defaults to A at position (0, 0) on canvas.
-let currentAtom: AtomNode = createAtom("A", new Point(0, 0));
+//AtomNode we are creating.
+let currentAtom: AtomNode;
```

Comment:
I liked how the old version gave us a default atom value
We should keep something like this in the new version.